bigtable/admin.go

```diff
@@ -1478,8 +1601,12 @@ func max(x, y int) int {
 //   and the given ClusterConfig.
 // - Any cluster missing from conf.Clusters but present in the instance will be removed from the instance
 //   using DeleteCluster.
-// - Any cluster in conf.Clusters that also exists in the instance will be updated to contain the
-//   provided number of nodes if set.
+// - Any cluster in conf.Clusters that also exists in the instance will be
+//   updated either to contain the provided number of nodes or to use the
+//   provided autoscaling config. If both the number of nodes and autoscaling
+//   are configured, autoscaling takes precedence. If the number of nodes is zero
```

Comment:
In this condition (zero and not provided), should we instead surface an error?

Reply:
It's a good question. The existing behavior was to ignore `NumNodes <= 0`, so if we change the code to return an error, we'd be changing the behavior. Since I'm new to the code here, I'm inclined to maintain the status quo and leave it for a future pass.

Reply:
sgtm!